packages/firefox-arm64-bin/firefox-arm64-bin.pacscript

```diff
@@ -7,10 +7,15 @@ description="Free and open-source web browser developed by the Mozilla Foundatio
 maintainer="Oren Klopfer <oren@taumoda.com>"
 hash="b197fe80bfe0c6025daf1364212ee1917c3781124b45545f41c7475f11c892d1"
 
+build() {
+    sudo mkdir -p "${pkgdir}/usr/lib/"
+    sudo mkdir -p "${pkgdir}/usr/lib/firefox"
+}
+
 install() {
     cd ..
     sudo install -Dm755 "usr/bin/firefox" -t "${pkgdir}/usr/bin/"
-    sudo cp -r "usr/lib/firefox/" "${pkgdir}/usr/lib/firefox"
+    sudo cp -r "usr/lib/firefox/*" "${pkgdir}/usr/lib/firefox/"
```

Comment:
```suggestion
    sudo cp -r "usr/lib/firefox/"* "${pkgdir}/usr/lib/firefox/"
```

Reply:
??
I am copying all of the files within the firefox folder to the new firefox folder; the * should be inside the quotes

Reply:
Did you test it?

Reply:
affirmative, the issue was with me not setting up the dir in pacscript